playground/terraform/infrastructure/appengine/main.tf

```diff
@@ -0,0 +1,67 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource "google_app_engine_application" "app_playground" {
+  project     = var.project_id
+  location_id = var.region == "us-central1" ? var.location_id_us : var.region
+  database_type = "CLOUD_DATASTORE_COMPATIBILITY"
+}
+
+resource "google_project_service" "firestore" {
+  project = var.project_id
+  service = "firestore.googleapis.com"
+  disable_dependent_services = true
+  depends_on = [
+    google_app_engine_application.app_playground
+    ]
+}
+
+resource "google_app_engine_flexible_app_version" "default_app" {
+  depends_on = [
+    google_app_engine_application.app_playground
+    ]
+  count      = var.create_default_service ? 1 : 0
+  service    = "default"
+  version_id = "mlflow-default"
+  runtime    = "custom"
+  project    = var.project_id
+
+  deployment {
+    container {
+      image = "gcr.io/cloudrun/hello"
```

Comment:
Fixed.

Reply:
Was forced to return back this option because of the following error during Frontend deployment:
Error: Error creating FlexibleAppVersion: googleapi: Error 400: The first service (module) you upload to a new application must be the 'default' service (module). Please upload a version of the 'default' service (module) before uploading a version for the 'frontend-stg' service (module). See the documentation for more information. Python: (https://developers.google.com/appengine/docs/python/modules/#Python_Uploading%%20modules) Java: (https://developers.google.com/appengine/docs/java/modules/#Java_Uploading%%20modules)